### PR TITLE
fix(ui5-input): allows changing value in selection change event handler

### PR DESCRIPTION
**Pull request by MapTo0**
FIXES: https://github.com/SAP/ui5-webcomponents/issues/5563

### CODE REVIEW

File: packages/main/src/Input.js [hunk 1]
```diff
@@ -826,7 +826,6 @@ class Input extends UI5Element {
 		}
 
 		if (!itemPressed) {
-			this.fireEventByAction(this.ACTION_ENTER, event);
```
Comment (MapTo0) [inline packages/main/src/Input.js:829]: @ilhan007 do you remember why we have fired enter manually? I did not find any place it is handled

**Comment by MapTo0** [comment]
FIXES: https://github.com/SAP/ui5-webcomponents/issues/5571

**Review by ivoplashkov — APPROVED** [review]
LGTM, waiting for reply from @ilhan007.

File: packages/main/src/Input.js (hunk 1 above)
Comment (ilhan007) [reply packages/main/src/Input.js:829]: Yes, indeed it's not handled anymore in fireEventByAction, it used to be handled for sure, but I don't quite remember why - I think it was related to IE, so if we want to find out what wash this about I would start looking in the changes related to the IE clean up, but I think we can just remove this line

**Review by ilhan007 — APPROVED** [review]

**Pull request merged**